testsuite/forge/src/backend/k8s/chaos/network_delay.yaml

```diff
@@ -0,0 +1,25 @@
+kind: NetworkChaos
```

Comment:
What are the thoughts on how can we have  a delay test that can inject delay as per a distribution - something like p90 delay of 1s and p50 delay of 300 ms?

Reply:
I'm not sure if we can be so exact with the distribution, but with the `correlation` and `jitter` configs for `latency` chaos, we should be able to get pretty close: https://chaos-mesh.org/docs/simulate-network-chaos-on-kubernetes/#delay